inc/commondbchild.class.php

```diff
@@ -452,25 +452,12 @@ function getHistoryNameForItem(CommonDBTM $item, $case) {
     * @return void
    **/
    function post_addItem() {
-      global $CFG_GLPI;
 
       $item = $this->getItem();
       if ($item === false) {
          return;
       }
 
-      if (in_array(static::class, $CFG_GLPI["infocom_types"], true) && in_array(static::$itemtype, $CFG_GLPI["infocom_types"], true)) {
```

Comment:
What about all other referenced `infocom_types`?

Reply:
hum, maybe none of them is a choomondbchild

Reply:
That's the point.